### PR TITLE
I18n - fix for #3846

### LINKED ISSUE #3846: Tooltip for Filament Settings/Cooling/Disable fan
### Version
1.3.0 dev

### Operating system type + version
Windows 10 1607

### Behavior
Tooltip for Filament Settings/Cooling/Disable fan for the first...________...layers
reads "You can set this to a positive value to disable fan at all during the first layers, so that is does not make adhesion worse. (default 3)"

Might be better worded.........
"This disables the fan completely for the first # of layers to aid in the adhesion of media to the bed. (default 3)"

### PR DESCRIPTION
**Pull request by ruggb**
this fixes #3846 - this is my very first attempt so check it out.......

### CODE REVIEW

File: translation/en_US.po [hunk 1]
```diff
@@ -285,11 +285,11 @@ msgstr "Disable fan for the first"
 
 #: xs/src/libslic3r/PrintConfig.cpp:169
 msgid ""
-"You can set this to a positive value to disable fan at all during the first "
-"layers, so that it does not make adhesion worse."
+"This disables the fan completely for the first # of layers "
```
Comment (lordofhyphens) [inline translation/en_US.po]: I think you should avoid using symbols, perhaps "first N layers" instead?

Comment (ruggb) [inline translation/en_US.po]: yea, I couldn't decide on that..........thx

**Comment by gege2b** [comment]
Hi @ruggb 
When doing the french translation, I used the word "plateau" to translate "bed", maybe you should use the same word to keep coherence ?

And I would suggest, for the first sentence : 
"Cela désactive complètement le ventilateur pour les N premières couches"

**Comment by lordofhyphens** [comment]
@gege2b I figured that translators made whatever editing choices as got the point across in their translations.  Not like I can tell the difference anyway ;) 

Feel free to update your translation in a separate PR if you think the French version should be updated (the English text, however, is the lookup key so any revisions to that text must be committed to the other translations).

I think this particular scenario highlights the need of having a separate repository that I can just give contributor access to translators.

**Comment by gege2b** [comment]
@lordofhyphens yep, I think you're right for the separate repo, it should make translation integration smoothier 

Right now, I have no git/poedit or whatever else available to make the changes myself (and can't install them). So it was easier and faster to tell the "needed" changes

**Comment by lordofhyphens** [comment]
It's all text, so even github webedit will do ;)

**Pull request merged**